src/Microsoft.AspNetCore.Mvc.Razor.Extensions/InjectDirective.cs

```diff
@@ -11,7 +11,10 @@ namespace Microsoft.AspNetCore.Mvc.Razor.Extensions
 {
     public static class InjectDirective
     {
-        public static readonly DirectiveDescriptor Directive = DirectiveDescriptorBuilder.Create("inject").AddType().AddMember().Build();
+        public static readonly DirectiveDescriptor Directive = DirectiveDescriptor.CreateDirective(
+            "inject",
+            DirectiveDescriptorKind.SingleLine,
+            builder => builder.AddTypeToken().AddMemberToken());
```

Comment:
There are still a few things here that don't feel good. 

I don't like that we put `descriptor` front and center. I want to call this class just `Directive`.

I want to make it easier to build a 'simple' directive.

Something like

`Directive.CreateSimpleDirective("inject", b=> b.AddTypeToken().AddMemberToken());`